src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -47,6 +70,7 @@ struct WhisperDecodedResultChunk {
 
 struct WhisperDecodedResults : public DecodedResults {
     std::optional<std::vector<WhisperDecodedResultChunk>> chunks = std::nullopt;
+    WhisperPerfMetrics perf_metrics;
```

Comment:
`DecodedResults ` already have `PerfMetrics perf_metrics`
how does it work that struct has 2 fields with the same name ?

Reply:
`WhisperDecodedResults` 'hides' parent's `PerfMetrics perf_metrics` member variable. After reading more on this topic, I guess it could be considered as questionable practice. I removed inheritance on `DecodedResults`.